org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ContentAssistNoTerminalExtensionTestLanguageGrammarAccess.java

```diff
@@ -83,132 +83,114 @@
 		private final Keyword cRpromoterRpromoterKeyword_19_0 = (Keyword)cRpromoterEnumLiteralDeclaration_19.eContents().get(0);
 		
 		//enum PolygonBasedNodeShape:
-		//	octagon | oval | parallelogram | pentagon | plain | plaintext | point | polygon | primersite | promoter |
-		//	proteasesite | proteinstab | rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter;
+		//    octagon | oval |
+		//    parallelogram | pentagon | plain | plaintext | point | polygon | primersite | promoter | proteasesite | proteinstab |
+		//    rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter;
 		public EnumRule getRule() { return rule; }
 		
-		//octagon | oval | parallelogram | pentagon | plain | plaintext | point | polygon | primersite | promoter |
-		//proteasesite | proteinstab | rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter
+		//octagon | oval |
+		//parallelogram | pentagon | plain | plaintext | point | polygon | primersite | promoter | proteasesite | proteinstab |
+		//rarrow | record | rect | rectangle | restrictionsite | ribosite | rnastab | rpromoter
 		public Alternatives getAlternatives() { return cAlternatives; }
 		
 		//octagon
 		public EnumLiteralDeclaration getOctagonEnumLiteralDeclaration_0() { return cOctagonEnumLiteralDeclaration_0; }
 		
-		//"octagon"
```

Comment:
We have a small loss of information here. Guess it is not really an issue. @szarnekow !?

Reply:
is here no node or dont we retrieve it correctly?

Reply:
There is no node. The string literal is optional and added during the Xtext linking.